### PR TITLE
Add post about code coverage for ruby on rails

### PR DESCRIPTION
**Pull request by etagwerker**
Hey, 

This is a post about Ruby on Rails's code coverage. It comes with a static website to show some of my research: https://fastruby.github.io/coverage/

~~WIP: I still need to re-calculated some of the coverage data for the first components. During the process of writing this article, I realized that I had been using the wrong SimpleCov code (it wasn't automatically merging results for different commands) 😢~~ 

~~Other than that,~~ this article is ready for review. 

Please check it out.

Thanks!

### CODE REVIEW

File: _posts/2020-08-23-what-is-code-coverage-ruby-on-rails.md [hunk 1]
```diff
@@ -0,0 +1,301 @@
...
+This is an article about my process and my findings.
+
+<!--more-->
+
+## Process
+
+In order to calculate code coverage, I used [SimpleCov](https://github.com/simplecov-ruby/simplecov) and 
+analyzed Rails at [f22dd39](https://github.com/rails/rails/commit/f22dd39cb2adf85d3deeca61f9465206f7bd8df3).
+
+I didn't run the entire test suite from Rails's root directory, I went into
```
[the remark this answers is not in the document]
Comment (rdormer) [reply _posts/2020-08-23-what-is-code-coverage-ruby-on-rails.md:26]: or just "root directory"

File: _posts/2020-08-23-what-is-code-coverage-ruby-on-rails.md [hunk 2]
```diff
@@ -0,0 +1,301 @@
...
+
+SimpleCov.start do
+  track_files '{lib}/**/*.rb' 
+  add_filter "/test/"
+end
+```
+
+I had to add the `SimpleCov.command_name` to make sure that all the test rake 
+tasks are considered and automatically merged by `SimpleCov`. Without that line 
+I was getting unexpected results when running more than one _test_ rake task (e.g.
```
[the remark this answers is not in the document]
Comment (etagwerker) [reply _posts/2020-08-23-what-is-code-coverage-ruby-on-rails.md:52]: no, I'll fix it

File: _posts/2020-09-01-what-is-code-coverage-ruby-on-rails.md [hunk 3]
```diff
@@ -0,0 +1,314 @@
...
+
+<!--more-->
+
+## Process
+
+In order to [calculate code coverage](https://www.fastruby.io/blog/rails/upgrades/simplecov/rails-upgrades-simplecov.html), 
+I used [SimpleCov](https://github.com/simplecov-ruby/simplecov) and 
+analyzed Rails at [f22dd39](https://github.com/rails/rails/commit/f22dd39cb2adf85d3deeca61f9465206f7bd8df3).
+
+<<<<<<< HEAD:_posts/2020-09-01-what-is-code-coverage-ruby-on-rails.md
```
Comment (cleicar) [inline _posts/2020-09-01-what-is-code-coverage-ruby-on-rails.md:27]: @etagwerker  This seems to be a git conflict, so I'm not sure which sentence to review. Could you fix them before I proceed?

Comment (etagwerker) [reply _posts/2020-09-01-what-is-code-coverage-ruby-on-rails.md]: @cleicar Just addressed your comment. Good catch! 💯

**Comment by etagwerker** [comment]
@lubc oops, done.

**Pull request merged**